src/util/ColorsUtil.js

```diff
@@ -0,0 +1,63 @@
+/*!
+ * Copyright (c) 2018-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(function () {
+  var fn = {};
+
+  var lighten = function (hex, lum) {
+    hex = String(hex).replace(/[^0-9a-f]/gi, '');
+    var newHex = '#';
+    for (var i = 0; i < 3; i++) {
+      var c = parseInt(hex.substr(i * 2, 2), 16);
+      c = Math.round(Math.min(Math.max(0, c + (c * lum)), 255)).toString(16);
+      newHex += ('00' + c).substr(c.length);
+    }
+    return newHex;
+  };
+
+  var template = function (colors) {
+    return `
+      #okta-sign-in.auth-container .button-primary,
```

Comment:
Looked at `@mixin button-template` and turns out it defines corresponding text color and border color.
Are we comfortable to omit both? Maybe Edbury could provide more help.

Reply:
I could add style for the border-color since in the `@mixin` it depends from primary button background color. 
However, the text color does not depend on the primary button background color, so I am not sure if we should include also that in this PR, since we may add a variable to control that later.
If we really want to generate also the primary button text color here, we could computationally generate a text color that has a good contrast with the primary button background color passed in config.

@nbarbettini @edburyenegren-okta opinions?

Reply:
We should do 2 things for a good developer experience:

1. For developers who want advanced customization of Widget look & feel, make it possible to modify the primary button text color as a Sass variable. @manueltanzi-okta can you create an issue to track this?

2. For developers who just want a quick & easy customization (and are not editing Sass), automatically make the primary button text color darker if the `colors.brand` color is very light.

Reply:
Created JIRA for SASS change: https://oktainc.atlassian.net/browse/OKTA-202207
Will add computed text color in this PR.

Reply:
As per discussion in SIW design meeting, we are going to skip the text color for now.